docs/rfcs/004-issueish-list.md

```diff
@@ -0,0 +1,98 @@
+# Issueish List
+
+## Status
+
+Proposed
+
+## Summary
+
+Display a list of all open pull requests in the current repository in the GitHub tab.
+
+## Motivation
+
+To provide a navigational element that makes sense even if you aren't on an active feature branch.
+
+To give users a way to see an overview of what's going on in the repository.
+
+As an initial building block toward a pull request review workflow.
+
+## Explanation
+
+### Accordion Lists
+
+Within the GitHub panel, render a vertical stack of two collapsible lists of _issueish_ (pull request or issue) items:
+
+_First list: current pull request_. If the active branch is associated with one or more open pull requests on the GitHub repository, render an item for each.
```

Comment:
In what case would it show more than one PR? Is if someone forks a repo, then makes a PR that has the same branch name as a PR that is on the upstream repo? Like

- `atom/atom:master` <- `atom/atom:feature`
- `atom/atom:master` <- `username/atom:feature`

Reply:
>  Is if someone forks use the same branch name and make a PR to your origin?

Yeah, that's the case where we show multiple now, like all of the "master" PRs it offers to let you pin.

> Couldn't this be reduced to just one if we would limit it to the current remote repo? Then there can only be one branch with the same name.

Which remote is the "current" one? Right now we just ask you for the "upstream" one.

This gets tricky when you have a network of multiple forks involved. I think you can send multiple pull requests from the same head ref to different base repositories in the same fork network. You might even be able to open multiple pull requests from the same head ref to different base refs within the same repository... ? I haven't tried it yet.

In any case, the by-far most common case is to have zero or one pull requests for a given head ref and head repository, but we've got a long tail of wacky edge cases where you _could_ have several. I figured that because we're already building a view based on stacked lists, we could handle them _and_ make it more consistent just by making the "current" section a list, too.

Reply:
> Which remote is the "current" one?

The one that it would use to pull/push. I think it's called "remote tracking branch". But I didn't consider that a local branch could have **no** remote tracking branch. Hmm.. maybe the title could say

- "Current PR" if there is just one
- "Related PRs" if there are more

Reply:
Oh, right - I can get that remote from your branch information. Well then :smile: